scripts/install.sh

```diff
@@ -1,12 +1,13 @@
-#!/bin/sh
+#!/bin/bash
 
+should_use_file_from_github_url=true
 # defaults
-PLACEHOLDER_REGEX_COMMIT_ISSUE_NUMBER="^\s*\[[\-\w]*\d\]"
+PLACEHOLDER_REGEX_COMMIT_ISSUE_NUMBER="^\s*[\-\w]*\d:"
 PLACEHOLDER_REGEX_BRANCH_ISSUE_NUMBER="[.]*\/([\-\w]*?\-\d+)"
 PLACEHOLDER_REGEX_GIT_COMMIT_MESSAGES="(Merge\sbranch\s\'|\#\sRebase\s|This\sreverts\scommit\s)"
 PLACEHOLDER_LOGGING_VERBOSE="true"
-
-GITHUB_SCRIPT_URL="https://raw.githubusercontent.com/janniks/prepare-commit-msg/master/scripts/prepare-commit-msg"
+PLACEHOLDER_NEW_COMMIT_MESSAGE="#{issue_number.upcase}: #{original_commit_message.gsub(/(\s[[:punct:]])+$/, '')}"
```

Comment:
I like the idea of parameterizing the new commit message. I can also add this to the install script 👍

Reply:
Thanks! That was actually the main reason why I forked your project